tools/lint.toml

```diff
@@ -23,18 +23,11 @@ target-version = "py39"
 "scipy/stats/tests/data/_mvt.py" = ["E701", "E702"]
 # E501 (line length) ignores
 "benchmarks/benchmarks/**" = ["E501"]
-"scipy/_lib/**" = ["E501"]
```

Comment:
I assume that the CI linter does not complain about violations in git submodules as I have never seen a fail for them.